libs/comments/website/src/lib/comment-list/comment-list-item.tsx

```diff
@@ -43,6 +44,8 @@ export const CommentListItemActionsButtons = styled('div')`
   justify-content: space-between;
 `
 
+export const CommentReplyButton = styled(Button)``
```

Comment:
This now breaks the website builder as we use a static button and not the one from the website builder FYI

Reply:
so how would you make it overridable without extracting to a new component?